torchtune/datasets/_grammar.py

```diff
@@ -19,6 +19,7 @@ def grammar_dataset(
     source: str = "liweili/c4_200m",
     column_map: Optional[Dict[str, str]] = None,
     train_on_input: bool = False,
+    system_prompt: Optional[str] = None,
```

Comment:
I think these need to be updated to or at least the documentation stating that this will override any system prompt?

Reply:
oi, missed these